articles/Frontend/Architecture/coding-guide.md

```diff
@@ -284,54 +279,136 @@ const testTaker = Object.assign(aUser, assignee);
 The particularity of this pattern is the scope is shared between the target and the mixin : `this` will be shared.
 There are multiple ways to achieve this pattern, including using prototypes.
 
-*When to use it ?*
+_When to use it ?_
 When mixins are pure methods or stateless by preference. There shouldn't be any strong coupling between the target and the mixin (for example the mixin expects a property to be available in the target).
 
 3. Delegation
 
-_TBD_
+Delegation pattern is composition pattern where component delegates functionality to other module
```

Comment:
**thought:** You could also mention the Adapter pattern. The API is defined by a façade which is delegating the call to a replaceable adapter containing the actual implementation.

Reply:
Added adapter pattern description and example